website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -0,0 +1,101 @@
+
+## Overview
+
+The motivations for the new layout are related to bringing more flexibility to users and 
+part of the process to [Integrating Kubebuilder and Operator SDK][integration-doc].
+
+### What was changed
+ 
+The `deploy` directory was replaced with the `config` directory including a new layout of Kubernetes manifests files
+- CRD's manifests in `deploy/crds/` are now in `config/crd/bases`
+- CR's manifests in `deploy/crds/` are now in `config/samples`
+- Controller manifest `deploy/operator.yaml` was replaced for `config/manager/manager.yaml` 
+- RBCA's manifests in `deploy` are in `config/rbac/`
+
+The `build/Dockerfile` directory was replaced by the `Dockerfile` in the root directory
+
+### What is new
+
+Projects are now scaffold using:
+
+- [kustomize][kustomize] to manage Kubernetes resources needed to deploy your operator
+- A `Makefile` with helpful targets for build, test, and deployment, and to give you flexibility to tailor things to your project's needs
+- Updated metrics configuration using [kube-auth-proxy][kube-auth-proxy], a `--metrics-addr` flag, and [kustomize][kustomize]-based deployment of a Kubernetes `Service` and prometheus operator `ServiceMonitor`
+
+## How to migrate
+
+The easy migration path is to create a new project from the scratch and let the tool scaffold the files properly and then, 
+just replace with your customizations and implementations. Following an example. 
+ 
+### Creating a new project
+
+In Kubebuilder-style projects, CRD groups are defined using two different flags
+(`--group` and `--domain`).
+
+When we initialize a new project, we need to specify the domain that _all_ APIs in
+our project will share, so before creating the new project, we need to determine which
+domain we're using for the APIs in our existing project.
+
+To determine the domain, look at the `spec.group` field in your CRDs in the
+`deploy/crds` directory.
+
+The domain is everything after the first DNS segment. Using `cache.example.com` as an
+example, the `--domain` would be `example.com`.
+
+So let's create a new project with the same domain (`example.com`):
+
+```sh
+$ mkdir memcached-operator
+$ cd memcached-operator
+$ operator-sdk init --plugins=ansible --domain=example.com
+```
+
+Now that we have our new project initialized, we need to re-create each of our APIs. 
+Using our API example from earlier (`cache.example.com`), we'll use `cache` for the
+`--group` flag.
+
+For each API in the existing project which is using a role, run:
+```sh
+$ operator-sdk create api \
+    --group=cache \
+    --api-version=<apiVersion> \
+    --kind=<Kind> 
+```
+
+**Note** Ensure that you use the same values for the flags to recreate the same Helm Chart and API's. If you have
+more than one chart or API's you can add them via `operator-sdk create api` command. For further information check the [quick-start][quickstart].
```

Comment:
Copy paste error (s/Helm/Ansible/), but I think this is already covered, maybe remove this note but leave a link.